README.md

```diff
@@ -41,8 +41,14 @@ PHIN VADS written in Go. Load `phinvads.dump` into a PostgreSQL database using `
     cd ..
     ```
 
-1. Run the app!
+1. Run the app! If you are only working on backend code, you can just run a simple live reload with air:
 
     ```bash
     air
     ```
+
+1. Air will also work for the frontend, but you will have to refresh your browser every time you make a change. To get automatic browser reloads, run the app this way:
+
+    ```bash
+    templ generate --watch --proxy="http://localhost:4000" --cmd="go run ./cmd/phinvads-go -tls=false"
```

Comment:
@katyasoup was that behavior in your comment just from running `air` or was it from this  ^ whole command?

Reply:
the `templ generate ...` from the readme